src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -142,29 +172,20 @@ const FailedToAddUsersMessage: React.FC<FailedToAddUsersMessageProps> = ({
       <div className="message-body">
         {isOpen && (
           <>
-            {Object.entries(groupedUsers).map(([domain, domainUsers]) => (
-              <p
-                key={domain}
-                data-uie-name="multi-user-not-added-details"
-                data-uie-value={domain}
-                style={{lineHeight: 'var(--line-height-sm)'}}
-              >
-                <span
-                  css={warning}
-                  dangerouslySetInnerHTML={{
-                    __html: t(`failedToAddParticipantsPluralDetails${errorMessageType}`, {
-                      name: domainUsers[domainUsers.length - 1].name(),
-                      names:
-                        domainUsers.length === 2
-                          ? domainUsers[0].name()
-                          : domainUsers.map(user => user.name()).join(', '),
-                      domain,
-                    }),
-                  }}
-                />
+            {message.reason === AddUsersFailureReasons.UNREACHABLE_BACKENDS && (
+              <>
+                {Object.entries(groupedUsers).map(([domain, domainUsers]) => (
+                  <MessageDetails key={domain} domain={domain} message={message} users={domainUsers}>
+                    {learnMore}
```

Comment:
I guess we should only show the `learnMore` once for all the backends (here it's going to show for every single backend line, I feel, right?)